integration/testcmd/dhclient/main.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2013-2019 the Webboot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"github.com/u-root/webboot/pkg/dhclient"
+)
+
+func main() {
+	//call pkg/dhclient's New() with these parameters 
+	// ^e.* = interface name 
+	// 15 = timeOut in seconds
+	// 5 = number of retries 
+	// false = verbose
+	// the last true's are for searching both ipv4 and ipv6
```

Comment:
you can take this comment block out. If people need to know what it is they can use go doc.

Reply:
I have removed this comment out. I realize go doc will help describe this action by typing in go doc in the terminal and it shows the user the various different functions, variables, and flags.